### PR TITLE
Transaction Sequence Validator

### PR DESCRIPTION
**Pull request by Solonets**
### Description of the Change

Add Validators for transaction sequence (batches) ->

### Benefits

Needed for batch logic

### Possible Drawbacks 
--

### CODE REVIEW

File: test/module/shared_model/builders/protobuf/transport_builder_test.cpp [hunk 1]
```diff
@@ -434,3 +469,110 @@ TEST_F(TransportBuilderTest, BlockVariantWithInvalidBlock) {
...
+  std::move(
+      std::begin(batch2), std::end(batch2), std::back_inserter(transactions));
+  transactions.push_back(createTransaction());
+  transactions.push_back(createTransaction());
+  transactions.push_back(createTransaction());
+  std::move(
+      std::begin(batch3), std::end(batch3), std::back_inserter(transactions));
+  transactions.push_back(createTransaction());
+  auto val = framework::expected::val(
+      TransportBuilder<interface::TransactionSequence,
```
Comment (kamilsa) [inline test/module/shared_model/builders/protobuf/transport_builder_test.cpp]: Please introduce using for transport builder

File: test/module/shared_model/builders/protobuf/transport_builder_test.cpp [hunk 2]
```diff
@@ -16,6 +16,8 @@
  */
 
 #include <gtest/gtest.h>
+#include <interfaces/iroha_internal/transaction_sequence.hpp>
```
Comment (kamilsa) [inline test/module/shared_model/builders/protobuf/transport_builder_test.cpp]: Fix brackets

File: test/module/shared_model/builders/protobuf/transport_builder_test.cpp [hunk 3]
```diff
@@ -16,6 +16,8 @@
  */
```
Comment (igor-egorov) [inline test/module/shared_model/builders/protobuf/transport_builder_test.cpp]: Please update file header

**Comment by igor-egorov** [comment]
Please merge develop branch into yours. That should fix failing tests.

**Pull request merged**